smart_contracts/contracts/tutorial/hello-world/Cargo.toml

```diff
@@ -2,15 +2,15 @@
 name = "hello-world"
 version = "0.1.0"
 authors = ["darthsiroftardis <karan@casperlabs.io>", "Michał Papierski <michal@casperlabs.io>"]
-edition = "2021"
-
-[dependencies]
-casper-contract = "1.4.4"
-casper-types = "1.5.0"
+edition = "2018"
```

Comment:
We're going back with edition?
_(also seen in other place below)_

Reply:
Well, I think these two smart contracts were maybe copied in from elsewhere and the edition came over with it.

Everywhere else we specify `edition = 2018`, so I think these should be aligned, and when we bump the edition it should be a deliberate decision and applied consistently everywhere in the repo.

Reply:
Makes sense, thanks for explanation.